### PR TITLE
add one liner to get started to docs

### LINKED ISSUE #1135: Add a "one liner" to the home page
Many OSS project sites include a quick command that someone can run to try the thing out. The foremost example of this in my mind is [Homebrew](https://brew.sh/) (see screenshot below)

Screen Shot 2019-03-14 at 11 52 52 AM

The command they give quickly gets you started and up and running.

I think we should have the `docker run` command featured prominently on the front page of our docs!

This is a follow-up on https://github.com/gomods/athens/pull/1082#pullrequestreview-214245155

### PR DESCRIPTION
**Pull request by cByst**
**What is the problem I am trying to address?**

Adding a one liner command to the docs to get new users up and running quickly.

**How is the fix applied?**

Added this one liner command to the main docs page.

**Mention the issue number it fixes or add the details of the changes if it doesn't have a specific issue.**

Fixes #1135

### CODE REVIEW

**Comment by LinuxSuRen** [comment]
I'm a little busy recently. But I'm sure to want to help with this.

File: docs/content/_index.es.md [hunk 1]
```diff
@@ -11,6 +11,13 @@ date: 2019-03-14T08:44:12+00:00
 
 En esta página web, está documentado Athens en detalle. Te enseñaremos que hace, por que es necesario, y como puedes ejecutarlo por tu cuenta. A continuación te mostramos un breve listado.
 
+#### ¿Cómo empezar?
+    Correr `docker run -p '3000:3000' gomods/athens:latest`
```
Comment (lopezator) [inline docs/content/_index.es.md]: Correr effectively means run, but only in the context of sport/physical exercise. It's better to use "Ejecutar" here, meaning "execute".

File: docs/content/_index.es.md [hunk 2]
```diff
@@ -11,6 +11,13 @@ date: 2019-03-14T08:44:12+00:00
...
+
+Luego, configura tu `GOPROXY` y `go get` go!
```
Comment (lopezator) [inline docs/content/_index.es.md]: Luego means later, I wouldn't use this one here.

Not sure about how to translate this word joke but...

"Ahora configura tu `GOPROXY` y a darle al `go get`!

Could work.

File: docs/content/_index.zh.md [hunk 3]
```diff
@@ -11,6 +11,13 @@ date: 2018-12-07T11:44:36+00:00
 
 在这个站点上，我们详细地记录了 Athens 的细节。我们将会告诉您它都做了什么，它为什么有意义，你可以用它来做什么，以及你如何运行它。下面是简单的概要。
 
+#### 如何开始？
+    运行 `docker run -p '3000:3000' gomods/athens:latest`
+
+然后，设置你的 `GOPROXY` 和 `go get` 去!
```
Comment (Xuanwo) [inline docs/content/_index.zh.md]:
```
然后设置你的 `GOPROXY` 并 `go get` 去！
```

`设置 A 和 B 去` means going to set for both A and B, and here we need to use `设置 A 并 B 去` which means set A first than execute B.

Comment (cByst) [reply docs/content/_index.zh.md]: @Xuanwo if you  have some time can you take a look at the change I made? I was a little confused but I think I made the change you were referring to.

Comment (Xuanwo) [reply docs/content/_index.zh.md]: @cByst Looks good to me now.

**Comment by lopezator** [comment]
LGTM

**Pull request merged**